### ISSUE TITLE
right scroll issue

### ISSUE THREAD

**Issue by sergeiandreyev** [ra3xdh/qucs_s#884]
Version: qucs s24.3.0 (12ddd12b)
OS: Ubuntu 22.04 LTS

When I scroll to the right in schematic window Qucs-S crashes with:
```
qucs-s: /home/andreev/opensource/qucs_s/qucs/schematic.cpp:2463: void Schematic::scrollRight(int): Assertion `step >= 0' failed.
Aborted (core dumped)
```

**Comment by wawuwo** [comment]
@sergeiandreyev, hi! I've prepared a fix for the problem

**Comment by ra3xdh** [comment]
I didn't understand how to reproduce this, but I have accepted #886 as the fix. Fixed by #886. Closing as completed.

**Comment by sergeiandreyev** [comment]
Thank you, @ra3xdh!
